org.mattbas.Glaxnimate.yml

```diff
@@ -0,0 +1,27 @@
+app-id: org.mattbas.Glaxnimate
+runtime: org.kde.Platform
+runtime-version: "6.3"
+sdk: org.kde.Sdk
+sdk-version: "6.3"
+command: glaxnimate
+finish-args:
+    - --share=ipc
+    - --share=network
+    - --socket=x11
+    - --socket=wayland
+    - --filesystem=host
```

Comment:
Does it use file portal? We discourage direct filesystem access like this if file picker can do the job.

Reply:
It can load files based on IPC commands, would the portal work in that case?

Reply:
I assume it wouldn't.